mailroom/drone_telemetry telemetry
mailroom/ATSCsignal[] signal
uint32 battery
